app/src/main/java/org/akvo/flow/util/image/GlideImageLoader.java

```diff
@@ -112,9 +112,34 @@ public void loadFromFile(File file, ImageView imageView) {
                 .into(imageView);
     }
 
+    @Override
+    public void loadFromFile(File file, ImageView imageView, DrawableLoadListener listener) {
+        requestManager.load(file)
+                .skipMemoryCache(true)
+                .diskCacheStrategy(DiskCacheStrategy.NONE)
+                    .listener(new RequestListener<Drawable>() {
+                        @Override
+                        public boolean onLoadFailed(@Nullable GlideException e,
+                                Object model, Target<Drawable> target, boolean isFirstResource) {
+                            listener.onLoadFailed();
+                            return false;
+                        }
+
+                        @Override
+                        public boolean onResourceReady(Drawable resource, Object model,
+                                Target<Drawable> target, DataSource dataSource,
+                                boolean isFirstResource) {
+                            listener.onLoadSucceeded();
+                            return false;
```

Comment:
do we need to return false in `onResourceReady` ?

Reply:
it's set to false because we update the image view ourselves because it needs some customization and not the library.